DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -22,14 +24,15 @@ public IActionResult Index()
                 return RedirectToAction("Index", "Login");
             }
 
-            var userEmail = User.GetCustomClaim(ClaimTypes.Email);
-            var delegateNumber = User.GetCustomClaim(CustomClaimTypes.LearnCandidateNumber);
             var centreId = User.GetCentreId();
-            var firstName = User.GetCustomClaim(CustomClaimTypes.UserForename);
-            var surname = User.GetCustomClaim(CustomClaimTypes.UserSurname);
             var centreName = centresService.GetCentreName(centreId);
 
-            var model = new MyAccountViewModel(centreName, userEmail, delegateNumber, firstName, surname);
+            var userAdminId = User.GetCustomClaim(CustomClaimTypes.UserAdminId);
+            var userDelegateId = User.GetCustomClaim(CustomClaimTypes.LearnCandidateId);
+            var (adminUser, delegateUser) = userService.GetUsersById(userAdminId, userDelegateId);
+
+            var model = new MyAccountViewModel(adminUser, delegateUser, centreName);

```

Comment:
I wonder if we even need the centre name anymore as it looks like we're joining with the centres table in the db queries when getting the user data..

Reply:
Yeah we shouldn't need to, particularly as I need to get more information off the centres for the third subtask